### PR TITLE
Warrior Agility Damage Nerf And A New Var

### PR DESCRIPTION
**Pull request by Isy232**
## About The Pull Request

Establishes the melee_dmg_mod for xenos and uses it to decrease the slash damage a warrior does while in agility mode by 60%.

## Why It's Good For The Game

Warrior using agility to deal damage is far too strong and this should make the extra agility fair.

## Changelog
:cl:
balance: Warrior Damage In Agility Reduced By 60%
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/mob/living/carbon/xenomorph/attack_alien.dm [hunk 1]
```diff
@@ -164,6 +164,9 @@
 	if(!can_xeno_slash(X))
 		return FALSE
 
+	// copypasted from attack_alien.dm
+	//From this point, we are certain a full attack will go out. Calculate damage and modifiers
+	var/damage = X.xeno_caste.melee_damage * (X.melee_dmg_mod)
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/xenomorph/attack_alien.dm]: Damage is being defined twice here.

File: code/modules/mob/living/carbon/xenomorph/attack_alien.dm [hunk 2]
```diff
@@ -164,14 +164,15 @@
...
 	if(!prob(X.melee_accuracy))
 		playsound(loc, 'sound/weapons/slashmiss.ogg', 25, TRUE)
 		X.visible_message("<span class='danger'>\The [X] slashes at [src], narroly missing!</span>",
 		"<span class='danger'>Our slash against [src] narroly misses!</span>")
 		return FALSE
 
-	var/damage = X.xeno_caste.melee_damage
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/xenomorph/attack_alien.dm]: `var/damage = X.xeno_caste.melee_damage * (X.melee_dmg_mod)`

File: code/modules/mob/living/carbon/xenomorph/attack_alien.dm (hunk 1 above)
Comment (Rohesie) [reply code/modules/mob/living/carbon/xenomorph/attack_alien.dm]: Remove this one

**Comment by Rohesie** [comment]
Changes pending

**Comment by spookydonut** [comment]
Use the attack signals to modify the damage when in agility mode.

File: code/modules/mob/living/carbon/xenomorph/attack_alien.dm [hunk 3]
```diff
@@ -166,13 +166,17 @@
...
+
+	var/attack_flick =  "slash"
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/attack_alien.dm]: your merge went badly

**Comment by spookydonut** [comment]
stale closing

**Pull request closed without merging**